docs/timeseries/masking.rst

```diff
@@ -1,23 +1,33 @@
 .. _timeseries-masking:
 
-Masking values in time series
+Masking Values in Time Series
 *****************************
 
 .. |Table| replace:: :class:`~astropy.table.Table`
+.. |TimeSeries| replace:: :class:`~astropy.timeseries.TimeSeries`
 
 .. warning:: Note that masking does not yet work for columns that have units.
 
+.. Is this warning up to date?
```

Comment:
Yes, unfortunately MaskedQuantity is not yet a thing, https://github.com/astropy/astropy/issues/1852